lib/aur-sync

```diff
@@ -205,9 +205,19 @@ tmp_view=$(mktemp -d --tmpdir "aurutils-$argv0-view.XXXXXXXX") || exit
 
 trap 'trap_exit' EXIT
 
+# Extract 'pkgname' from command-line.
+unset pkgname
+while true; do
```

Comment:
```suggestion
while (( $# )) ; do
```
Otherwise when processing `pkgname` as the last argument, on the next loop `shift` will exit with error causing the `sync` to terminate as it has `set -o errexit`

Reply:
Thanks.